src/ansys/units/quantity.py

```diff
@@ -78,6 +78,8 @@ def __init__(
             raise InsufficientArguments()
 
         if not isinstance(value, (float, int)):
+            if isinstance(value, str):
+                raise ValueError("value should be either float, int or [float, int].")
```

Comment:
Should this be a TypeError?

Reply:
No. Incorrect value is passed not the value type.

Reply:
But we are checking type of `value`.

https://docs.python.org/3/library/exceptions.html#TypeError

Reply:
@mkundu1 Right, ValueError will be raised for correct type but incorrect value. It should be TypeError.

![image](https://github.com/ansys/pyansys-units/assets/106588300/2b2c4f1b-f4a6-48c8-afae-a1a2416ee354)

Reply:
Done. Thank you.